.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
   "trailingComma": "es5",
   "tabWidth": 2,
-  "singleQuote": true
+  "singleQuote": true,
+  "endOfLine": "auto"
```

Comment:
can we use lf instead of auto?

Reply:
We could be we have to rewrite all new lines probably..

<img width="783" alt="image" src="https://github.com/wiktor-obrebski/type-safe-errors/assets/3736993/0de4e65c-3de2-43e6-8aaa-8c6fb94cfd66">